src/bertconf_bert_loader.erl

```diff
@@ -5,7 +5,7 @@
          code_change/3, terminate/2]).
 
 -include("bertconf.hrl").
--record(state, {watches, changes=[], old_tables=[], reloader=undefined}).
+-record(state, {ref, watches, changes=[], old_tables=[], reloader=undefined, needs_reload=false}).
```

Comment:
Yeah it's probably a better name. Note that this name was chosen because it's what was used in the previous version of the code: https://github.com/tokenrove/bertconf/blob/c7281994c7b75b2f8d23736fa3934a025e1eaa51/src/bertconf_bert_loader.erl#L8